src/schema/alter-table-builder.ts

```diff
@@ -224,6 +228,25 @@ export class AlterTableBuilder implements ColumnAlteringInterface {
     })
   }
 
+  addIndex(indexName: string): AlterTableAddIndexBuilder {
+    return new AlterTableAddIndexBuilder({
+      ...this.#props,
+      indexBuilder: new AddIndexBuilder({
+        ...this.#props,
+        node: AddIndexNode.create({ name: IdentifierNode.create(indexName) }),
+      }),
+    })
```

Comment:
I don't understand why both `AlterTableAddIndexBuilder` and `AddIndexBuilder` exist. Mind explaining? 😕